packages/ui/src/app/pages/WorkingGroups/WorkingGroup/WorkingGroup.stories.tsx

```diff
@@ -20,12 +25,46 @@ const WG_DATA = {
   name: 'membership',
 }
 
+const WG_OPENING_DATA = {
+  title: 'Membership worker role',
+  shortDescription: 'Lorem Ipsum...',
+  description: 'Bigger Lorem ipsum...',
+  applicationDetails: 'Application process default',
+  applicationFormQuestions: [
+    { question: '🐁?', type: OpeningMetadata.ApplicationFormQuestion.InputType.TEXT },
+    { question: '🐘?', type: OpeningMetadata.ApplicationFormQuestion.InputType.TEXTAREA },
+  ],
+  hiringLimit: 1,
+  expectedEndingTimestamp: 2000,
+}
+
+const WG_JSON_OPENING = {
+  title: 'Membership worker role',
+  shortDescription: 'Lorem Ipsum...',
+  description: 'Bigger Lorem ipsum...',
+  applicationDetails: 'Application process default',
+  applicationFormQuestions: [
+    { question: '🐁?', type: 'TEXT' },
+    { question: '🐘?', type: 'TEXTAREA' },
+  ],
+  hiringLimit: 5,
+  expectedEndingTimestamp: 2000,
```

Comment:
3. Re-use the metadata here so it's clear they have the same value:
```suggestion
  ...WG_OPENING_METADATA,
```